ui/components/app/permission-page-container/index.scss

```diff
@@ -94,6 +94,7 @@
     display: flex;
     flex-direction: column;
     margin-top: 38px;
+    width: 100%;
```

Comment:
Not entirely sure why this was needed :thinking: But if it is needed, we can split it out.

Reply:
@hmalik88